FinancialTracker/Models/User.swift

```diff
@@ -45,10 +44,9 @@ extension User: Codable {
         score = try values.decode(Double.self, forKey: .score)
         premium = try values.decode(Bool.self, forKey: .premium)
         FCMToken = try values.decodeIfPresent(String.self, forKey: .FCMToken) ?? ""
-        reminders = try values.decode([Reminder].self, forKey: .reminders)
     }
     
     enum CodingKeys: String, CodingKey {
-        case firstName, lastName, email, uid, balance, currency, expenses, score, premium, incomes, FCMToken, reminders
+        case firstName, lastName, email, uid, balance, currency, expenses, score, premium, incomes, FCMToken
```

Comment:
So you really need the `CodingKeys`?

Reply:
I use them

Reply:
Ok for now. 
FYI, there is a `FirebaseFirestoreSwift` SPM package that you can use to save yourself the manual data mapping from/to firebase firestore. 

See https://peterfriese.dev/posts/firestore-codable-the-comprehensive-guide/ - we can do this after the March 30th deadline.